packages/cli/src/utils/calculate-retry-delay.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Calculates the delay for the next retry attempt using exponential backoff with full jitter.
+ *
+ * @param attempt The current retry attempt number (starting from 0 for the first retry).
+ * @param baseDelay The initial delay in milliseconds (e.g., 100).
+ * @param maxDelay The maximum possible delay in milliseconds (e.g., 20000).
+ * @returns The calculated delay in milliseconds to wait before the next attempt.
+ */
+export function calculateRetryDelay(
```

Comment:
question: As this logic already lives in the MAPI client, and the CLI is leveraging it, is it needed to repeat this logic at this level? (same with `delay.ts`)

Reply:
Yeah, it's not using the client for the login: https://github.com/storyblok/monoblok/blob/bugfix/cli-migrations-retry/packages/cli/src/commands/login/actions.ts#L38

I don't like the repetition either, but I did want to use the same algo in both cases, yet I don't think it is warranted to introduce a new package for shared code just for that. But maybe we should think about it for the future.